package-lock.json

```diff
@@ -9,6 +9,7 @@
       "version": "1.0.0",
       "license": "MIT",
       "dependencies": {
+        "@ministryofjustice/frontend": "^3.0.3",
```

Comment:
This has automatically moved out to the main dependencies from the Dev dependencies. Not sure why.